Source/Fuse.Nodes/Transform.uno

```diff
@@ -29,8 +29,17 @@ namespace Fuse
 		
 		internal event Action<Transform> MatrixChanged;
 
+		bool _wasFlat;
```

Comment:
Yes, I know, but a Transform may not be flat by default. Some of the constructors take arguments, don't they?